### DISCUSSION TITLE
Adjust display duration of skip button

### DISCUSSION THREAD

**Discussion by pafade89** [intro-skipper/intro-skipper#421] [Q&A] [answered]
Hi,

is it now possible to adjust the display duration of the skip button? It often disappears too quickly. I saw in #34 that it wasn’t possible 11 months ago. Has anything changed, or will it change in the future? Or is it simply not possible via the plugin?

Thank you, and have a great evening!

**Comment by AbandonedCart** [comment, marked as answer]
Jellyfin 10.10 brought a button provided by the clients. This means the plugin no longer has control over the duration it is displayed, how it looks, etc. etc. The plugin only provides the start and end times of the specific areas that will be skipped.

**Reply by pafade89** [reply]
Thank you for your quick response.

**Comment by jumoog** [comment]
I have noticed that each client has a different duration. Some show the button much longer than others.
